magda-sleuther-framework/src/index.ts

```diff
@@ -148,7 +148,7 @@ export default async function sleuther(
             }
         }).map((page: RecordsPage<Record>) => page.records);
 
-        await forEachAsync(registryPage, 20, (record: Record) =>
+        await forEachAsync(registryPage, 1, (record: Record) =>
```

Comment:
It can be a separate PR, but this seems like it should be a parameter to the `sleuther` function.  Some sleuthers will benefit from being able to process more than one record at a time.

Reply:
Agreed.